src/common/constants.ts

```diff
@@ -142,6 +142,16 @@ export class Constants {
     public static amlWorkspaceDesc = "Azure Machine Learning Workspace";
     public static asaJobDesc = "Azure Stream Analytics Job";
     public static amlApiVersion: string = "2018-03-01-preview";
+
+    public static needSimulatorInstalledMsg = "You must have the 'iotedgehubdev' tool installed for Edge Hub simulation.";

```

Comment:
`Edge Hub simulation` -> `IoT Edge simulator` for alignment and avoiding introducing new concept?